### DISCUSSION TITLE
How do I turn a manually created table into a collection I can see/modify in the web portal?

### DISCUSSION THREAD

**Discussion by g1sbi** [pocketbase/pocketbase#2531] [Q&A]
I'm using a separate server, which gathers data and, given a database, creates tables and stores it. I successfully had it create the tables inside the pocketbase's database, but I can't see them as collections. Ideally, I'd like to see these tables in the portal such that I can easily control and modify the entries. 

I don't know if this is  the easiest to achieve it but I thought of creating the tables myself as collections first, and modify the external server's database model to fit pocketbase's collection requirements? Would that work?

These are the models:
```
interface ThreadsInstance extends Model {
  id: number;
  website: string;
  title: string;
  start: Date;
  scraped: Date;
  updated: Date;
  topic: number;
  author: string;
}
```
```
interface ImagesInstance extends Model {
  id: number;
  thread_id: number;
  name: string;
  url: string;
  is_hidden: boolean;
  sort_order: number;
}
```
```
interface VendorsInstance extends Model {
  id: number;
  thread_id: number;
  location: string;
  url: string;
}
```

**Comment by ganigeorgiev** [comment]
> I thought of creating the tables myself as collections first, and modify the external server's database model to fit pocketbase's collection requirements? Would that work?

Yes, it should work. Eventually in the future we may add support for autodiscovery of plain tables but for now this has very low priority.
